source/gui/settingsDialogs.py

```diff
@@ -268,31 +275,35 @@ class SettingsPanel(wx.Panel, DpiScalingHelperMixin, metaclass=guiHelper.SIPABCM
 	title=""
 	panelDescription=u""
 
-	def __init__(self, parent):
+	def __init__(self, parent: wx.Window):
 		"""
 		@param parent: The parent for this panel; C{None} for no parent.
-		@type parent: wx.Window
 		"""
 		if gui._isDebug():
 			startTime = time.time()
 		wx.Panel.__init__(self, parent, wx.ID_ANY)
 		DpiScalingHelperMixin.__init__(self, self.GetHandle())
 
+		self._buildGui()
+
+		if gui._isDebug():
+			elapsedSeconds = time.time() - startTime
+			panelName = self.__class__.__qualname__
+			log.debug(f"Loading {panelName} took {elapsedSeconds:.2f} seconds")
+
+	def _buildGui(self):
 		self.mainSizer=wx.BoxSizer(wx.VERTICAL)
 		self.settingsSizer=wx.BoxSizer(wx.VERTICAL)
 		self.makeSettings(self.settingsSizer)
-		self.mainSizer.Add(self.settingsSizer, flag=wx.ALL)
+		self.mainSizer.Add(self.settingsSizer, flag=wx.ALL | wx.EXPAND)
 		self.mainSizer.Fit(self)
 		self.SetSizer(self.mainSizer)
-		if gui._isDebug():
-			log.debug("Loading %s took %.2f seconds"%(self.__class__.__name__, time.time() - startTime))
 
 	@abstractmethod
-	def makeSettings(self, sizer):
+	def makeSettings(self, sizer) -> wx.BoxSizer:

```

Comment:
I think this annotation is incorrect.

```suggestion
	def makeSettings(self, sizer: wx.BoxSizer) -> None:
```